dnf-behave-tests/dnf/repolist.feature

```diff
@@ -21,8 +22,9 @@ Scenario: Repolist without arguments
     And stdout does not contain "dnf-ci-thirdparty\s+dnf-ci-thirdparty"
 
 
+@dnfdaemon
 Scenario: Repolist with "enabled"
-   When I execute dnf with args "repolist enabled"
+   When I execute dnf with args "repolist --enabled"
```

Comment:
I'd like to preserve also original scenario ("repolist enabled"). Although we do not want this syntax supported in dnf5/dnfdaemon, it's still supported in dnf4.
We can turn this scenario into `Scenario Outline` to test both possibilities (enabled / --enabled) and only tag `--enabled` variant with `@dnfdaemon`:

```
Scenario Outline: Repolist with "<argument>"
   When I execute dnf with args "repolist <argument>"
   ...
Examples: positional argument for yum compatibility
    | argument      |                     
    | enabled       |                     
@dnfdaemon
Examples: named argument
    | argument      |
    | --enabled     |
```

Or create a new scenario with `--enabled` which is probably better readable.